component/src/main/java/io/siddhi/extension/io/kafka/source/KafkaSource.java

```diff
@@ -116,20 +115,21 @@
                         type = {DataType.STRING},
                         optional = true,
                         defaultValue = "null"),
-                @Parameter(name = "enable.auto.commit",
-                        description = "This parameter specifies whether to commit offsets automatically. \n"
+                @Parameter(name = "enable.offset.commit",
+                        description = "This parameter specifies whether to commit offsets. \n"
```

Comment:
@pcnfernando shall we rearrange the parameter description as follows?

This parameter specifies whether to commit offsets.

If the manual offset committing is needed, `auto.commit.interval.ms` should be `false` and `enable.offset.commit` should be `true`.
If periodical committing is needed, `auto.commit.interval.ms` should be `true` and `enable.offset.commit` should be `true`.
If committing is not needed, `auto.commit.interval.ms` should be `false`.

Note: 'enable.auto.commit' is an optional property in the Kafka. If `enable.auto.commit` property is set to `true`, it will periodically (Configurable with `auto.commit.interval.ms` property as an `optional.configuration` as well default value is 1000ms.) commits offsets. This will disabled when `enable.offset.commit` is set to `false`.

Reply:
Addressed in 148bf3d7aac73f320be0e826999538ca83ec62d0